src/lib/interactionManager/guards.ts

```diff
@@ -23,4 +27,15 @@ export const isCredentialOfferResponse = (
 ): token is CredentialOfferResponse => token instanceof CredentialOfferResponse
 
 export const isCredentialReceive = (token: any): token is CredentialsReceive =>
-  token instanceof CredentialsReceive
\ No newline at end of file
+  token instanceof CredentialsReceive
+
+export const isAuthorizationRequest = (
+  token: any,
+): token is AuthorizationRequest =>
+  !!(token as AuthorizationRequest).imageURL &&
```

Comment:
This does not look very reliable. Can we use `instanceOf` instead?

Reply:
I dont think so, because they're interfaces not classes